packages/defi-sdk/resolvers/token-resolvers/resolvers/ethereum/src/schema.graphql

```diff
@@ -0,0 +1,8 @@
+#import { Module, Connection } into Ethereum from "wrap://ens/ethereum.polywrap.eth"
+#import { Module } into TokenResolver from "wrap://ens/interface.token.resolvers.defiwrapper.eth"
+
+type Module implements TokenResolver_Module {}
+
+type Env {
+  connection: Ethereum_Connection!
+}
```

Comment:
Remove Env from here since not needed. ethereum plugin support env now and we can configure the env for ethereum plugin from client now.